services/brig/src/Brig/API/User.hs

```diff
@@ -201,7 +202,9 @@ createUser new@NewUser {..} = do
             { newUserManagedBy = case mbExistingAccount of
                 Nothing -> newUserManagedBy
                 Just acc -> Just . userManagedBy . accountUser $ acc,
-              newUserIdentity = ident
+              newUserIdentity = case mbExistingAccount of
```

Comment:
Yes, if I consult my reflog, the commit that introduced the change had message.  

> Fix:createUser ignores user identity of invitation

I don't remember the details.
I reverted this change (https://github.com/wireapp/wire-server/pull/1393/commits/0e152da188b2ed18306e952538f8b8b59f9fa51f) because all tests still pass without it.

Reply:
this may be due to my changing the definition of `ident` a few lines above?

Reply:
No, weirdly enough not. I reverted the `ident` definition and the tests still passed. It must've been some other changes.